test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -5,8 +5,8 @@ const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
 
-const fn = path.join(common.fixturesDir, 'elipses.txt');
-const rangeFile = path.join(common.fixturesDir, 'x.txt');
+const fn = path.join(common.fixtures, 'elipses.txt');
```

Comment:
This one would fail tests as `common` module does not export fixtures. Have a look at `test/common/fixtures.js` module and use `fixtures.path`.

Reply:
Correct, the change here should be:

```js
const fixtures = require('../common/fixtures');
```

then...

```js
const fn = fixtures.path('elipses.txt');
const rangeFile = fixtures.path('x.txt');
```